client/App.tsx

```diff
@@ -37,14 +41,15 @@ const App = (): ReactElement => {
   const handleTestClick = function (): void {
     console.table(tracks);
     console.log(genres);
+    fetchTracks();
   };
 
 
 
   return (
     <>
       <h1>DJ Database</h1>
-      <button onClick={handleTestClick}>TEST</button>
+      <button onClick={(handleTestClick)}>TEST</button>
```

Comment:
Is there a reason these parentheses were added here?

Reply:
My only guess is a new extension I installed in VSCode changing it. Trying out extensions as they're suggested in my class, I end up turning off 80% of them.